app/models/preserved_object.rb

```diff
@@ -26,4 +26,19 @@ def self.normalize_druid_name
       po.save!
     end
   end
+
+  def create_archive_copies(archive_vers)
```

Comment:
any chance this would be misinterpreted as "it's going to actually do the replication" ?     Add a comment?  Rename?  Or just let folks read the code?

Other naming suggestions:  `create_archive_preserved_copies` ?